metafix/src/main/java/org/metafacture/metafix/api/FixFunction.java

```diff
@@ -24,13 +24,20 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.function.Consumer;
 import java.util.stream.Stream;
 
 @FunctionalInterface
 public interface FixFunction {
 
     void apply(Metafix metafix, Record record, List<String> params, Map<String, String> options);
 
+    default void getOption(final Map<String, String> options, final String key, final Consumer<String> consumer) {
```

Comment:
Maybe something like `consumeOption` or `getAndConsumeOption` would be clearer? (`getOption` is fine with me though).

Reply:
Yeah, I wasn't really sure about it myself. Naming things is hard ;)

Originally, I had `withOption()`. But then I opted for alignment with the existing `getBoolean()`/`getInteger()` instead.

Reply:
> Originally, I had `withOption()`.

I like that, reads good: `withOption ... setFormat`

Reply:
Okay, I've changed the name in 710db71.

Reply:
:+1: